templates/sponsors/partials/full_sponsorship.txt

```diff
@@ -11,7 +11,9 @@
 # Sponsorship Information
   * Status		{{ sponsorship.status }}
 {% if sponsorship.level_name %}  * Level		{{ sponsorship.level_name }}{% else %}  * Level			Custom Package{% endif %}
-{% if for_modified_package %}  * Sponsorship Fee	To be determined{% else %}  * Sponsorship Fee	${{ sponsorship.sponsorship_fee|intcomma }} USD{% endif %}
-  * Benefits
-{% for benefit in  benefits %}    - {{ benefit.program.name }} - {{ benefit.name }}
-{% endfor %}
+{% if not display_fee %}  * Sponsorship Fee	To be determined{% else %}  * Sponsorship Fee	${{ sponsorship.sponsorship_fee|intcomma }} USD{% endif %}
```

Comment:
I realized that the new address fields are missing herein the `full_sponsorship.txt` =S

Reply:
Fixed in 
be6273c